dao/src/main/java/com/epam/esm/entity/GiftCertificate.java

```diff
@@ -0,0 +1,117 @@
+package com.epam.esm.entity;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+import java.util.StringJoiner;
+
+/**
+ * GiftCertificate entity.
+ *
+ * @since 1.0
+ */
+public class GiftCertificate {
+
+    private Long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private Integer duration;
+    private String createDate;
+    private String lastUpdateDate;
```

Comment:
Why createDate and lastUpdateDate has String type?
Acceptance Criteria: CreateDate, LastUpdateDate - format ISO 8601 (https://en.wikipedia.org/wiki/ISO_8601). Example: 2018-08-29T06:12:15.156. More discussion here: https://stackoverflow.com/questions/3914404/how-to-get-current-moment-in-iso-8601-format-with-date-hour-and-minute

Reply:
Date -fields is now stored in db as DATETIME format, In application as LocalDateTime.
The setting up of the fields is handled by database. 
commit fa7af67